rules-reviewed/springboot/tests/springboot.rhamt.test.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<ruletest id="ocp-001-test"
```

Comment:
Change the test file consistently with the changes applied to the `springboot.rhamt.xml` file